packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/CollectionItemScopeHandler/CollectionItemScopeHandler.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Direction,
+  Position,
+  TextEditor,
+  type ScopeType,
+} from "@cursorless/common";
+import type { LanguageDefinitions } from "../../../../languages/LanguageDefinitions";
+import { BaseScopeHandler } from "../BaseScopeHandler";
+import { compareTargetScopes } from "../compareTargetScopes";
+import { TargetScope } from "../scope.types";
+import { ScopeIteratorRequirements } from "../scopeHandler.types";
+import { createTargetScope } from "../SurroundingPairScopeHandler/createTargetScope";
+import { getDelimiterOccurrences } from "../SurroundingPairScopeHandler/getDelimiterOccurrences";
+import { getIndividualDelimiters } from "../SurroundingPairScopeHandler/getIndividualDelimiters";
+import {
+  IndividualDelimiter,
+  IndividualSeparator,
+} from "../SurroundingPairScopeHandler/types";
+
+export class CollectionItemScopeHandler extends BaseScopeHandler {
+  public readonly iterationScopeType: ScopeType = {
+    type: "oneOf",
+    scopeTypes: [
+      { type: "line" },
+      {
+        type: "surroundingPairInterior",
+        delimiter: "any",
+      },
+    ],
+  };
```

Comment:
Actually this shouldn't be `oneOf`, it should be what I think we call `fallback`, where it uses inside if it can find it and only uses line if it finds no containing inside